config/schema_config.yaml

```diff
@@ -191,6 +191,16 @@ super enhancer:
   is_a: non coding element
   inherit_properties: true
 
+non coding rna:
+  represented_as: node
+  input_label: non_coding_rna
+  is_a: non coding element
+  inherit_properties: true
+  properties:
+    rna_type: str
+    rfam: str[]
+    go_term: str[]
```

Comment:
I think we should have a new relationship in the schema since `go gene` and `go gene product` are desired for gene. but in this dataset it provides annotation for `rna`. 
I have put sample schema below.
```
go rna:  
  is_a: annotation
  inherit_properties: true
  represented_as: edge
  input_label: belongs_to
  source: rna
  target: go
  properties:
```

Reply:
The above sample looks good.